Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -536,6 +536,15 @@ try {
         Copy-Item -Path $buildOutputFile -Destination $destFolder -Force -ErrorAction SilentlyContinue
         Copy-Item -Path $containerEventLogFile -Destination $destFolder -Force -ErrorAction SilentlyContinue
     }
+
+    & $PSScriptRoot\..\CheckForNewWarnings\CheckForNewWarnings.ps1 `
```

Comment:
The CheckForNewWarnings is placed in the Actions area even though it isn't an action.
We also typically don't call functions by calling a script this way - we normally declare and call functions (I am aware we do it the other way in actions)

I think the code in CheckForNewWarnings.ps1 should be moved into the psm1 module - the module should be moved to the RunPipeline action folder, loaded and then the module imported and called here if we are to check for new warnings.